pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -346,6 +347,10 @@ type VolumeToMount struct {
 	// ReportedInUse indicates that the volume was successfully added to the
 	// VolumesInUse field in the node's status.
 	ReportedInUse bool
+
+	// DesiredSizeLimit indicates the desired upper bound on the size of the volume
+	// (if so implemented)
+	DesiredSizeLimit *resource.Quantity
```

Comment:
sorry to comment the late. But where this DesiredSizeLimit is used?

Reply:
It will be used when we implement quota enforcement.

Reply:
shouldn't we hold from introducing this until we have a plan to use it? It seems like we introduced this field and almost an year later nobody is using it.